### ISSUE TITLE
false positive from `no-unused-imports` when importing an inner declaration with wildcard

### ISSUE THREAD

**Issue by RBusarow** [pinterest/ktlint#1393]
Given this file:

```kotlin
package com.example

import com.example.Outer.*

class Outer {
  class Inner
}

val foo = Inner()
```

command:
```
ktlint <path-to-file> --disabled_rules=no-wildcard-imports
```

Ktlint incorrectly says that `import com.example.Outer.*` is unnecessary.  If the wildcard is replaced with the explicit import, Ktlint no longer complains.

## Your Environment
* Version of ktlint used: 0.43.2 and 0.44.0
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): cli

**Comment by paul-dingemans** [comment]
Sorry, commit above is in no way related to this issue.
